src/mappings/erc-7589/role-granted-handler.ts

```diff
@@ -0,0 +1,57 @@
+import { log } from '@graphprotocol/graph-ts'
+import { generateTokenCommitmentId, upsertRoleAssignment } from '../../utils/helpers/erc-7589'
+import { TokenCommitment } from '../../../generated/schema'
+import { RoleGranted } from '../../../generated/AavegotchiDiamond/AavegotchiDiamond'
+import { getOrCreateUser } from '../../utils/helpers/diamond'
+
+/** 
+@dev This handler is called when a role is granted.
+@param event RoleGranted The event emitted by the contract.
+
+Example:
+     event RoleGranted(
+        uint256 indexed _commitmentId,
+        bytes32 indexed _role,
+        address indexed _grantee,
+        uint64 _expirationDate,
+        bool _revocable,
+        bytes _data
+    );
+*/
+export function handleRoleGranted(event: RoleGranted): void {
+  const commitmentId = event.params._commitmentId
+  const rolesRegistryAddress = event.address.toHexString()
+  const tokenCommitmentId = generateTokenCommitmentId(rolesRegistryAddress, commitmentId)
+  const tokenCommitment = TokenCommitment.load(tokenCommitmentId)
+
+  if (!tokenCommitment) {
+    log.error('[erc-7589][handleRoleGranted] TokenCommitment {} not found, tx hash: {}', [
+      tokenCommitmentId,
+      event.transaction.hash.toHexString(),
+    ])
+    return
+  }
+
+  const grantor = getOrCreateUser(tokenCommitment.grantor)
```

Comment:
`findOrInsert` is more commonly used
```suggestion
  const grantor = findOrInsertUser(tokenCommitment.grantor)
```

Reply:
`getOrCreateUser` is a function that was already in the repo and it was created by aavegotchi team. I don't know if would be a good idea to change the function name